packages/ui/components/form/wizard/wizardForm.test.tsx

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable playwright/missing-playwright-await */
+import { render, waitFor } from "@testing-library/react";
+import { vi } from "vitest";
+
+import WizardForm from "./WizardForm";
+
+vi.mock("next/navigation", () => ({
+  useRouter() {
+    return { replace: vi.fn() };
+  },
+  useSearchParams() {
+    return { get: vi.fn().mockReturnValue(currentStepNavigation) };
+  },
+}));
+
+const steps = [
+  {
+    title: "Step 1",
+    description: "Description 1",
+    content: <p data-testid="content-1">Step 1</p>,
+    isEnabled: false,
+  },
+  {
+    title: "Step 2",
+    description: "Description 2",
+    content: (setIsLoading: (value: boolean) => void) => (
+      <button data-testid="content-2" onClick={() => setIsLoading(true)}>
+        Test
+      </button>
+    ),
+    isEnabled: true,
+  },
+  { title: "Step 3", description: "Description 3", content: <p data-testid="content-3">Step 3</p> },
+];
+
+const props = {
+  href: "/test/mock",
+  steps: steps,
+  nextLabel: "Next step",
+  prevLabel: "Previous step",
+  finishLabel: "Finish",
+};
+
+let currentStepNavigation: number;
+
+const renderComponent = (extraProps?: { disableNavigation: boolean }) =>
+  render(<WizardForm {...props} {...extraProps} />);
+
+describe("Tests for WizardForm component", () => {
+  test("Should render view of component correctly", async () => {
+    currentStepNavigation = 1;
+    const { getByTestId, queryByText } = renderComponent();
+    const { nextLabel } = props;
+
+    expect(getByTestId("wizard-form")).toBeInTheDocument();
+    expect(getByTestId("step-title")).toBeInTheDocument();
+    expect(getByTestId("step-description")).toBeInTheDocument();
+    expect(getByTestId("content-1")).toBeInTheDocument();
+    expect(queryByText(nextLabel)).toBeInTheDocument();
```

Comment:
Isn't this redundant? Queries are the methods that Testing Library gives you to find elements on the page. The assertion 'toBeInTheDocument' does the same thing.